src/jit/codegencommon.cpp

```diff
@@ -5412,22 +5410,39 @@ void CodeGen::genAllocLclFrame(unsigned  frameSize,
 #else // !CPU_LOAD_STORE_ARCH
 
         // loop:
```

Comment:
Can you add: "Code size for each instruction. We need this because the backwards branch is hard-coded with the number of bytes to branch."